Marlin/configuration_store.cpp

```diff
@@ -58,7 +58,7 @@
  *  188  M206 XYZ  home_offset (float x3)
  *
  * Mesh bed leveling:
- *  200  M420 S    active (bool)
+ *  200  M420 S    status (0 = empty, 1 = has mesh numbers)
```

Comment:
This PR would have been merged a while ago, but it's still awaiting action. Rather than keep waiting, I will make the suggested changes myself, create a new PR, and merge it from there.